cfgov/v1/management/commands/invalidate_page_cache.py

```diff
@@ -4,15 +4,16 @@
 
 
 class Command(BaseCommand):
-    help = "Invalidate the cache of a page by its full URL"
+    help = "Invalidate the cache of pages by full URLs"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "--url",
             required=True,
-            help="The full URL for the page you wish to invalidate its cache",
+            help="The full URLs for pages you wish to invalidate its cache",
```

Comment:
```suggestion
            nargs="+",
            help=(
                "The full URL for the page cache to invalidate "
                "(can specify multiple)"
            )
```

Reply:
Updated `help` and used `nargs` to simplify.